### PR TITLE
Fix conesearch tutorial

### PR DESCRIPTION
**Pull request by adrn**
Fix Vizier URLs, à la https://github.com/astropy/astroquery/pull/1102/files

### CODE REVIEW

File: tutorials/notebooks/vo/conesearch.ipynb [hunk 1]
```diff
@@ -141,7 +137,7 @@
    "source": [
     "# Now use your favorite text editor to open the my_result.tex file.\n",
     "# For example:\n",
-    "!more my_result.tex"
+    "!cat my_result.tex"
```
Comment (pllim) [inline tutorials/notebooks/vo/conesearch.ipynb]: Just curious, does your machine not have `more`?

**Comment by pllim** [comment]
Close #229 

LGTM. Thanks!

File: tutorials/notebooks/vo/conesearch.ipynb (hunk 1 above)
Comment (adrn) [reply tutorials/notebooks/vo/conesearch.ipynb]: On my machine, `more` never lets the cell finish executing, so subsequent cells never run.

Comment (pllim) [reply tutorials/notebooks/vo/conesearch.ipynb]: :scream: ok

**Comment by adrn** [comment]
Merging this because travis passed. The circleci build is experimental, and is being worked on in a separate PR (#228)

**Pull request merged**